packages/react/modules/components/configure-adapter/configure-adapter.tsx

```diff
@@ -20,18 +20,14 @@ import AdapterContext, { createAdapterContext } from '../adapter-context';
 
 type valueof<T> = T[keyof T];
```

Comment:
Type utilities should be written with capital letter: 

```suggestion
type ValueOf<T> = T[keyof T];
```